components/MovieListing/MovieListing.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import { settings } from "../../common/apis/settings";
+import Slider from "react-slick";
+import { getAllMovies, getAllShows } from "../../features/movies/movieSlice";
+import MovieCard from "../MovieCard/MovieCard";
+import "./MovieListing.scss";
+const MovieListing = () => {
+  
+  const movies = useSelector(getAllMovies);
+  const shows = useSelector(getAllShows);
+  let renderMovies,
+    renderShows = "";
+
+  renderMovies =
+    movies.Response === "True" ? (
+      movies.Search.map((movie, index) => (
+        <MovieCard key={index} data={movie} />
+      ))
+    ) : (
+      <div className="movies-error">
+        <h3>{movies.Error}</h3>
+      </div>
+    );
+
+  renderShows =
+    shows.Response === "True" ? (

```

Comment:
object naming need to be changed  . inside object usually wont start with capital letters .  rather than checking for string as 'True ' you can maintain that as boolean